### PR TITLE
[8.x] Support Laravel 11

### PR DESCRIPTION
**Pull request by joostdebruijn**
Laravel 11 will be released in March, this PR prepares this package for Laravel 11. It also:

- Drops support for PHP-versions below 8.1, because they are not supported anymore
- Drops support for Laravel-versions below 9 (Laravel 9 itself is also EOL, but that allows developers on that version to prepare upgrades to a later version)
- Uses the latest version of Testbench and thus is running with PHPunit 10
- It drops support for the 3.x version of Money, because that version doesn't support PHP 8.0 and upwards
- It bumps the Github Action-dependencies to the latest stable version

It think this one should be merged into the next major release of this package.

### CODE REVIEW

**Comment by joostdebruijn** [comment]
@ricardogobbosouza Could you please merge and release this one?

**Comment by joostdebruijn** [comment]
Thanks for kicking-off the tests. I thought they were running well, however I'll have a look at it!

**Comment by joostdebruijn** [comment]
@ricardogobbosouza They should run fine now with the latest commit, see similar issue description here: https://github.com/orchestral/testbench-core/discussions/194

File: phpunit.xml.dist [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
+      <clover outputFile="clover.xml"/>
+      <text outputFile="php://stdout"/>
+    </report>
+  </coverage>
+  <source>
+    <include>
+      <directory suffix=".php">./src</directory>
+    </include>
+  </source>
 </phpunit>
```
Comment (ricardogobbosouza) [inline phpunit.xml.dist:19]: What has really changed here?

Comment (joostdebruijn) [reply phpunit.xml.dist:19]: I just ran the phpunit --migrate-configuration command to align the XML with the new PHPunit XML schema. Doesn't have functional impact.

**Pull request merged**